src/platforms/python/guides/rq/index.mdx

```diff
@@ -31,6 +31,24 @@ The integration will automatically report errors from all RQ jobs.
 
 Generally, make sure that the **call to `init` is loaded on worker startup**, and not only in the module where your jobs are defined. Otherwise, the initialization happens too late and events might end up not being reported.
 
+In addition, make sure that **`init` is called once** in your app and imported from there in the different places it is needed. In a Flask app, for example, Sentry and `RqIntegration` can be initalized as 
+
+```python {filename:app.py}
+import sentry_sdk
+from sentry_sdk.integrations.flask import FlaskIntegration
+from sentry_sdk.integrations.rq import RqIntegration
+
+sentry = sentry_sdk.init(
+    dsn=___PUBLIC_DSN___,
+    integrations=[FlaskIntegration(), RqIntegration()])
```

Comment:
Hmm, in this case, is the `RqIntegration` used in the Flask app too?

If the Flask app and Rq job have different integration needs, it might be a better suggestion to have independent calls to `sentry_sdk.init` -- in https://github.com/getsentry/sentry-python/issues/907 the problem was that the worker never initialized the Sentry SDK.

Reply:
Good point. 

I tried initialising it in my worker but the messages and exceptions were not sent _unless_ I added a breakpoint in the queued function and stepped through it line by line. My hunch was that something was off with the initialisation (order) and https://github.com/getsentry/sentry-python/issues/935 seems to confirm that. Hence my decision to initialise it in Flask.

Reply:
I can also open an issue about this if you prefer.

Reply:
I see, so does your worker import the Flask app?

The advice there makes sense in that case. In this short example I'm afraid it could be misleading for other people.

Reply:
Yes, it does (although I set `sentry = sentry_sdk.init()` and import it in my worker i.e. I use it as I'd initially documented it).

I'll leave it to you to decide whether to include this or not. It took me a while to get it to work and wanted to save a fellow dev's time. :) 

The solution presents itself fairly quickly if one browses `sentry-python` repo's closed issues but I started from `rq`'s docs, searched Sentry's forum and only then looked in the issue tracker.

Reply:
Please do not ever ask people to have two separate init calls. It is extremely confusing if for whatever reason both modules get imported and one init call overrides the other.